pkg/core/disk.go

```diff
@@ -23,8 +25,9 @@ const (
 
 type DiskCommand struct {
 	Action          string `json:"action"`
-	Size            uint64 `json:"size"`
+	Size            string `json:"size"`
```

Comment:
why change the size from int to string?

Reply:
Because size will read from string, and used as string (sprint). The another reason is I want a empty string as default value means None size , but not a 0 value in u64.

Reply:
OK